traits.yml

```
@@ -575,9 +575,9 @@ specifications:
             name: Entity
           - namespace: audio
             name: Audio
-      SampledAudio:
+      SampledAudioResource:
         description: >
-          Any entity that holds time-sampled audio data in a sidecar
+          Any entity that holds time-sampled audio data in a independent
```

[consider] Here and elsewhere during the change from sidecar->independent
```suggestion
          Any entity that holds time-sampled audio data in an independent
```